lib/views/swagger_v1.yml.ejs

```diff
@@ -1232,6 +1232,15 @@ paths:
         corresponding places
       parameters:
         - $ref: "#/parameters/path_multi_id_or_slug"
+        - name: admin_level
```

Comment:
Should we also make this an enum since we know only 6 values will be supported right now?

Reply:
Changed to enum but since it can be an array of values the interface still shows it without list of enumerated values:
![image](https://user-images.githubusercontent.com/16937734/105489467-effe6b00-5cbb-11eb-955a-b2c03be71bbf.png)